docs/project-docs/developer-guide.md

```diff
@@ -66,3 +66,8 @@ One can build in Debug or Release mode from the root by doing `build.cmd -Releas
 
 We only support 64-bit binaries right now.
 
+### Updating manifest and ep-list files
+
+During development, there may arise a need to update the current baseline `core_manifest.json` and `core_ep-list.tsv` files.
```

Comment:
When this is needed, what does the error say? (good for search-ability, and letting users know this is the right fix for their problem)